cmake/std/atdm/ats2/environment.sh

```diff
@@ -0,0 +1,37 @@
+################################################################################
+#
+# Set up env on ats2 for ATMD builds of Trilinos
+#
+# This source script gets the settings from the ATDM_CONFIG_BUILD_NAME var.
+#
+################################################################################
+
+# ats2 jobs all use the same environmnet changes to the
+# sourced script below will impact jobs on both of those
+# machines. please be mindful of this when making changes
+
+if [[ "$ATDM_CONFIG_KOKKOS_ARCH" == "DEFAULT" ]] ; then
```

Comment:
As we discussed, copy how `KOKKOS_ARCH` is handled for waterman at:

* https://github.com/trilinos/Trilinos/blob/develop/cmake/std/atdm/waterman/environment.sh#L38

You can almost copy and paste that if/else/fi block of code.